task_3.py

```diff
@@ -0,0 +1,21 @@
+"""Introduction to programming’: Task 3,
+   Kyshynska Taisiia"""
+import math as m
+
+print('Introduction to programming’: Task 2')
+print('Kyshynska Taisiia')
+
+TEMPLATE = '{} = '
+x = float(input(TEMPLATE.format('x')))
+z = float(input(TEMPLATE.format('z')))
+
+if isinstance(x, (int, float)) and isinstance(z, (int, float)): # для проверки того, что переменные x и z являются числами,
```

Comment:
there are only two possible flows of execution: 
1. input is a number so the variables contain float data_type.
2. input is not a number so the program fails during cast the variables to float.
As a result, the **if** is redundant

Reply:
but It's cool that you've played with **isinstance**

Reply:
but stop use cyrilic in your code

Reply:
but I like that you added comments